octomap_server/src/OctomapServer.cpp

```diff
@@ -1061,6 +1061,10 @@ void OctomapServer::reconfigureCallback(octomap_server::OctomapServerConfig& con
     m_groundFilterAngle         = config.ground_filter_angle;
     m_groundFilterPlaneDistance = config.ground_filter_plane_distance;
     m_maxRange                  = config.sensor_model_max_range;
+    m_probHit                   = config.sensor_model_hit;
+    m_probMiss                  = config.sensor_model_miss;
+    m_thresMin                  = config.sensor_model_min;
+    m_thresMax                  = config.sensor_model_max;
```

Comment:
The probabilities and thresholds are not used from these member variables but need to explicitly set to the m_octree object (see "initialize" starting at 104). With this in mind I can see that it's confusing to have them as members at all, instead of only in a local scope here and in the constructor.